### ISSUE TITLE
ASN and LX Reuse Facility should default to enabled

### ISSUE THREAD

**Issue by Fish-Git** [Fish-Git/hyperion#30]
Hercules's facility table (`FACTAB` in `archlvl.c`) currently has the _**ASN-and-LX Reuse Facility**_ defaulting to _disabled_:
```
#if defined(_FEATURE_ASN_AND_LX_REUSE)
FACILITY( ASN_LX_REUSE,  0, /*Z390*/   NONE,   Z390,   ALS2|ALS3 )
#endif
```
thereby requiring users to insert an `ARCHLVL ENABLE ASN_LX_REUSE` statement into their Hercules configuration file when running an operating system that expects the facility to be present. (Otherwise such operating systems won't even IPL.)

The historical reason(s) for why it was decided to default the facility bit to off (disabled) is unimportant. What _is_ important (IMO) is to correct the error by always defaulting to _enabled_, thereby reversing the requirement for our users to _instead_ insert a `ARCHLVL DISABLE ASN_LX_REUSE` statement into their Hercules configuration file when running an operating system that is _incompatible_ with the facility (which are less common than those expecting it).

This issue is just a reminder ("TO-DO" item) so I can remember to change it and to give others an opportunity to comment on this issue before it is implemented.

**Comment by Fish-Git** [comment]
This issue has already been fixed in my private Bitbucket repository and will eventually appear in my public GitHub repository just as soon as I'm finished with what I'm currently working on.

**Comment by Fish-Git** [comment]
Fixed in commit 4ee084f9e958ff5ae88c3ccad8be7500be47ebc5
